roadmap.md

```diff
@@ -54,33 +34,29 @@ a set of [library](/libs) major versions. Each library follows
 idom](https://en.cppreference.com/w/cpp/language/pimpl). This development
 and release pattern allows us to distribute patch and minor updates into an already released Ignition version. For example, if a new feature does not break API/ABI then we will target the feature to the oldest compatible non-EOL release and propagate the feature forward.
 
-### Blueprint
-
-1. Additional graphical tools for model and world creation and editing.
-1. Logical audio sensor and source.
-
 ### Citadel
 
 1. SDF website update, more documentation.
 1. Ease migration of SDF files from Gazebo classic.
-1. [Reset world.](https://github.com/ignitionrobotics/ign-gazebo/issues/203)
 
 ### Dome
 
-1. Particle effects, to support smoke and gas.
+1. [Particle effects on Gazebo, to support smoke and gas.](https://github.com/ignitionrobotics/ign-gazebo/issues/558)
 1. Localized wind (wind that is constrained to a region of influence).
-1. Optical tactile plugin.
 1. [Improved Ignition Physics documentation.](https://community.gazebosim.org/t/gsod-2020-ignition-physics-tutorial-and-api-documentation/746)
 1. [Bazel build files.](https://github.com/ignitionrobotics/ign-bazel)
+1. [Configure lights from the GUI or transport.](https://github.com/ignitionrobotics/ign-gazebo/issues/122)
```

Comment:
You could add a line about an improved thermal camera: https://github.com/ignitionrobotics/ign-gazebo/pull/498